### ISSUE TITLE
How to get 24bit True Colors to work on macOS? (mosh 1.3.2)

### ISSUE THREAD

**Issue by norman-thomas** [mobile-shell/mosh#945]
Hi,
sorry for this, if it's just too obvious.
I've been trying to get the true colors to work with mosh, but somehow it always fails for me.

I use mosh 1.3.2 for both client and server. On macOS (10.13.1) I tried both the HomeBrew and compiled version. For the server I tried both macOS and Ubuntu 16.04 (with self-compiled mosh 1.3.2).

True Colors work flawlessly via SSH:

<img width="1242" alt="screen shot 2017-11-23 at 11 07 40" src="https://user-images.githubusercontent.com/5961249/33167881-491f8460-d03f-11e7-95e8-932a9841880d.png">

Using mosh 24bit colors are broken while 256 colors work:

<img width="1102" alt="screen shot 2017-11-23 at 11 21 27" src="https://user-images.githubusercontent.com/5961249/33168232-7ee5bbea-d040-11e7-8e03-9e3a0e0acd60.png">

I used [this 24bit color test script](https://raw.githubusercontent.com/JohnMorales/dotfiles/master/colors/24-bit-color.sh).

I did read through #649 and #939.
Am I missing any additional configuration needed to make this work?

To sum up my environment:
- mosh 1.3.2
- macOS 10.13.1 + Ubuntu 16.04
- iTerm 3.1.5 with `xterm-256color` setting
- zsh on both client and server

**Comment by cgull** [comment]
True color support is not in 1.3.2, or any release, yet.  You need to install Mosh from Git master, on both client and server.  Fortunately this is really easy for both macOS and Ubuntu, use `brew install --HEAD mosh` and the Ubuntu [PPAs](https://launchpad.net/~keithw/+archive/ubuntu/mosh-dev).

**Comment by norman-thomas** [comment]
@cgull Thanks for the help!

And sorry about the ticket, I thought 1.3.2 already included this feature.
